src/js/wall/ResetForm.tsx

```diff
@@ -0,0 +1,41 @@
+import { get } from "lodash-es";
+import React from "react";
+import { useForm } from "react-hook-form";
+import { connect } from "react-redux";
+import { resetPassword } from "../account/actions";
+import { InputError, InputGroup, InputLabel, InputSimple } from "../base";
+import { WallButton } from "./Container";
+
+export function ResetForm({ error, onReset, resetCode }) {
+    const { register, handleSubmit } = useForm({ defaultValues: { password: "" } });
```

Comment:
Needs docstring. Parameter typing can be skipped for now, since they will be fully changed in the next few PRs